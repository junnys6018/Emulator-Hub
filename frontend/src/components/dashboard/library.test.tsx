import React from 'react';
import renderer from 'react-test-renderer';
import Library from './library';

test('<Library /> renders null', () => {
    const tree = renderer.create(<Library console="CHIP 8">{null}</Library>).toJSON();
    expect(tree).toMatchSnapshot();
});

test('<Library /> renders children', () => {
    const tree = renderer
        .create(
            <Library console="CHIP 8">
                <span>child 1</span>
                <span>child 2</span>
            </Library>,
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});
