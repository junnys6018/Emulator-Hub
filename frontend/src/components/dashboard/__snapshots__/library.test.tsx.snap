// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Library /> renders children 1`] = `
Array [
  <h2
    className="text-xl mb-4 pt-2.5"
    id="chip-8"
  >
    CHIP 8
  </h2>,
  <div
    className=" mb-6 md:mb-12 grid gap-3 md:gap-9 auto-rows-auto grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 "
  >
    <span>
      child 1
    </span>
    <span>
      child 2
    </span>
  </div>,
]
`;

exports[`<Library /> renders null 1`] = `null`;
